examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1938,7 +1938,7 @@ class GLTFDracoMeshCompressionExtension {
 
 					resolve( geometry );
 
-				}, threeAttributeMap, attributeTypeMap );
+				}, threeAttributeMap, attributeTypeMap, undefined, reject );
```

Comment:
How about we assign `LinearSRGBColorSpace` instead of `undefined`?

Reply:
Brilliant, just pushed a commit to do just that! Thank you